components/script/script_module.rs

```diff
@@ -1343,6 +1346,7 @@ impl FetchResponseListener for ModuleContext {
                     false,
                     CanGc::note(),
                     self.introduction_type,
+                    1
```

Comment:
```suggestion
                    1 // line_number
```

It might also be worth leaving a comment explaining why a line number of 1 is a reasonable value.